### PR TITLE
doec/trans_md_2_rst (AIS-1025)

### PR DESCRIPTION
**Pull request by lijunru-hub**
1.将docs下的md文档修改为rst文档
2.修改了部分文档的格式

### CODE REVIEW

**Comment by 007-Natasha** [comment]
@lijunru-hub , pls move the "img" subfolder one level up. So we don't have the same images twice in different folders.

File: docs/en/audio_front_end/Espressif_Microphone_Design_Guidelines.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
+Espressif Microphone Design Guidelines
+======================================
+
+   This document provides microphone design guidelines and suggestions
```
Comment (007-Natasha) [inline docs/en/audio_front_end/Espressif_Microphone_Design_Guidelines.rst:4]: Remove the unnecessary indentation

File: docs/en/audio_front_end/README.rst [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+-  Speech recognition scenarios
+
+.. figure:: ../img/AFE_SR_workflow.png
+   :alt: overview
+
+   overview
+
+The workflow is as follows:
+
+1) Use **ESP_AFE_SR_HANDLE** to create and initialize AFE
```
Comment (007-Natasha) [inline docs/en/audio_front_end/README.rst:47]: Replace 1) to 1.

Comment (lijunru-hub) [reply docs/en/audio_front_end/README.rst:47]: done

File: docs/en/audio_front_end/Espressif_Microphone_Design_Guidelines.rst (hunk 1 above)
Comment (lijunru-hub) [reply docs/en/audio_front_end/Espressif_Microphone_Design_Guidelines.rst:4]: done

**Review by feizi — APPROVED** [review]

**Comment by lijunru-hub** [comment]
AFE 部分和 word 内容进行比较 review done

**Comment by lijunru-hub** [comment]
wake word engine 下的内容review done

**Comment by lijunru-hub** [comment]
> @feizi 需要将 图片：WakeNet_model , MultiNet_model 转化为代码生成的表格

done

**Pull request closed without merging**